src/akkudoktoreos/class_optimize.py

```diff
@@ -331,7 +331,7 @@ def optimierung_ems(
             "discharge_hours_bin": discharge_hours_bin,
```

Comment:
instead of this dict the structure could be described with a pydantic model which can be dumped to json. It is also capable of converting numpy arrays.